setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup
+
+setup(
+   name='planprogenerator',
+   version='1.0',
+   description='A simple toolkit to generate planpro files',
+   author='DRSS 2022',
+   packages=['planprogenerator', 'planprogenerator.model', 'planprogenerator.planproxml'],  # would be the same as name
```

Comment:
Hmm... I think we should not publish the `planproxml` package. Is there any reason for that?

Reply:
I think this is needed, since it is a dependency of the generator. If I exclude this package, then it is not part of the published package and causes an exception that the module planproxml could not be found.